tests/pending/run/t8199.scala

```diff
@@ -39,8 +39,10 @@ object Test extends dotty.runtime.LegacyApp {
     checkClassName(c.getName)
   }
   def checkClassName(name: String): Unit = {
-    val defaultMaxClassFileLength = 255
-    assert((name + ".class").length <= defaultMaxClassFileLength, name)
+    final val MaxSuffixLength = 7 // "$.class".length // potential module class suffix and file extension
+    final val MaxNameLength   = 240 - MaxSuffixLength
+
+    assert((name + ".class").length <= defaultMaxNameLength, name)
```

Comment:
Reading more carefully, `s/defaultMaxNameLength/MaxSuffixLength/`... this code isn't compiled yet (it's a pending test case) but let's keep it intact.

Reply:
` 
    val defaultMaxClassFileLength = 240
    assert((name + ".class").length <= defaultMaxClassFileLength, name)`

This makes more sense to match. Pushing modification in a sec